sites/public/pages/eligibility/age.tsx

```diff
@@ -46,7 +58,7 @@ const EligibilityAge = () => {
             <div className="form-card__pager-row primary">
               <Button
                 styleType={AppearanceStyleType.primary}
-                onClick={() => router.push('/eligibility/disability')}
+                onClick={() => router.push("/eligibility/disability")}
```

Comment:
Could we use the Eligibility Sections array to build this URL? it'd be more DRY

Reply:
Done.